test/unit/api-config-test.js

```diff
@@ -123,12 +125,11 @@ describe('API Config', function() {
                 active: '#FFFFFF',
                 background: '#000000'
             };
+            const config = new Config({ skin: skinObject });
 
-            let x = new Config({ skin: skinObject });
-            expect(x.skinUrl).to.equal(skinObject.url);
-            expect(x.skinColorInactive).to.equal(skinObject.inactive);
-            expect(x.skinColorActive).to.equal(skinObject.active);
-            expect(x.skinColorBackground, skinObject.background);
+            expect(config.skinUrl).to.equal(skinObject.url);
+            expect(config.skin).to.equal(skinObject.name);
+            expect(typeof config.skinColors).to.equal('object');
```

Comment:
This is unsafe. (eg. it would evaluate to `true` if `config.skinColors === null`).